### DISCUSSION TITLE
《现代C++白皮书》一周目读后感

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#300] [Announcements]
https://imcbc.cn/202209/cpp_hopl4_reading/?code=b665f15c211f7f37980f

**Comment by penndu** [comment]
跟C有关的都恐惧！

**Comment by caibingcheng** [comment]
> @penndu
> 跟C有关的都恐惧！

我学别的就不容易学好，比如objective-c，因为我脑子里都是C/C++这样想法，不太容易切换
